tests/integration/targets/s3_lifecycle/tasks/main.yml

```diff
@@ -180,6 +180,30 @@
         prefix: /something
       register: output
 
+    - assert:
+        that:
+          - output is not changed
+  # ============================================================
+    - name: Create a lifecycle policy, with onezone_ia and transition_days to 0
+      s3_lifecycle:
+        name: '{{ bucket_name }}'
+        transition_days: 0
+        storage_class: onezone_ia
```

Comment:
> An error occurred (InvalidArgument) when calling the PutBucketLifecycleConfiguration operation: 'Days' in Transition action must be greater than or equal to 30 for storageClass 'ONEZONE_IA'

you must use another storage class to test `transition_days: 0`. The same for the 2nd idempotency test.

Reply:
@fmenabe